rust/src/api/get_node_by_path.rs

```diff
@@ -1,10 +1,22 @@
-use crate::{_auto_generated, node::model::Node};
+use crate::node::model::Node;
+
+#[cfg(feature = "const")]
+use crate::_auto_generated;
+
+#[cfg(feature = "storage")]
+use crate::api::storage;
 
 #[cfg(feature = "const")]
 pub fn get_node_by_path(path: &str) -> Option<&Node> {
     _auto_generated::data::get_node_by_path(path)
 }
 
+#[cfg(feature = "storage")]
+pub fn get_node_by_path(data_path: &'static str, path: &str) -> Result<Node, std::io::Error> {
```

Comment:
the `data_path` being static, looks like it should be configured somewhere once and never set again, i suggest removing it since we do not implement the builder pattern here, and let the user provide the full path every time.

```suggestion
pub fn get_node_by_path(path: impl AsRef<Path>) -> Result<Node, std::io::Error> {
```